satpy/tests/writer_tests/test_utils.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2019 PyTroll developers
+
+# Author(s):
+
+#   Stephan Finkensieper <stephan.finkensieper@dwd.de>
```

Comment:
I'd like if we stopped putting individual authors in module headers. Additionally the above copyright should say "Satpy developers". I need to document this somewhere.

Reply:
Yeah agree, I just copied it from another file.